app/models/ect_at_school_period.rb

```diff
@@ -59,6 +59,7 @@ class ECTAtSchoolPeriod < ApplicationRecord
     with_expressions_of_interest_for_contract_period(year)
     .where(expression_of_interest: { lead_provider_id: })
   }
+  scope :active, -> { where(withdrawn_by_error: false) }
```

Comment:
Thanks Leandro, I guess my concern with only including this in siblings and thus the overlap validation is ignored, is that the rest of the scopes will return this record especially where we do not want it to be returned or included.

Maybe it would be good to try the default scope, where it is ignored and since we have to include it in the API how that would look like in terms of metadata and other queries that matter, happy to chat